### PR TITLE
Make copyright year in docs change with the current year

### PR DESCRIPTION
**Pull request by sarah256**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -22,12 +23,17 @@
 
 # -- Project information -----------------------------------------------------
 
+# Get current year for copyright
+t = datetime.today()
+dt = datetime(t.year, 1, 1)
+year = dt.strftime("%Y")
```
Comment (mprahl) [inline docs/conf.py]: You can simplify these three lines to:
```python
year = datetime.today().strftime('%Y')
```

File: docs/conf.py [hunk 2]
```diff
@@ -22,12 +23,17 @@
...
+
 try:
     version = pkg_resources.get_distribution('estuary').version
 except pkg_resources.DistributionNotFound:
     version = 'unknown'
 project = u'Estuary API'
-copyright = u'2018, Matt Prahl, Yash Nanavati, Sarah Rieger'
+copyright = year + u', Matt Prahl, Yash Nanavati, Sarah Rieger'
```
Comment (mprahl) [inline docs/conf.py]: FYI the `u'` isn't needed since you import `from __future__ import unicode_literals`. It doesn't hurt to have them though.

Comment (sarah256) [reply docs/conf.py]: @mprahl should i delete all of the u's then?

Comment (mprahl) [reply docs/conf.py]: You don't have to. If you want to, you can do that in a separate PR. It certainly looks prettier without them. :)

Comment (sarah256) [reply docs/conf.py]: okay will do! otherwise, this should be ready to go then :smile:

**Pull request merged**